my-app/src/components/atoms/appLogo/AppLogo.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Rename this to `LinkWithImage` 

To use it as app logo, simply pass the relevant props to this component